Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -421,6 +439,9 @@ - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)
 
 
 - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)isReferrable explicitlyRequestedReferrable:(BOOL)explicitlyRequestedReferrable automaticallyDisplayController:(BOOL)automaticallyDisplayController {
+    // Log this early. Not consistently showing up when logged from [Branch initialize].
+    [self.class addBranchSDKVersionToCrashlyticsReport];
```

Comment:
This was not consistently showing up in reports when I logged it in `[Branch initialize]`. This is a little mysterious. In Debug builds, I put a log message inside `[BNCCrashlyticsWrapper setObjectValue:forKey:]` and verified that `self.crashlytics` was non-nil there. But I could not get symbolicated reports for Debug builds because of missing dSYMs I couldn't find (should be under DerivedData). In testing I was working with Release builds distributed through Crashlytics but then... I didn't get log messages. The earlier these things are called, the better. But this was the only thing that worked consistently during testing. Maybe we can sort that out.